airbyte-integrations/connectors/destination-jdbc/src/test/java/io/airbyte/integrations/destination/jdbc/JdbcDestinationTest.java

```diff
@@ -1,335 +0,0 @@
-/*
```

Comment:
I decided that the test here were of very little use since:
1) they are almost an exact duplicate of the JDBC Integration tests in terms of what is being tested.
2) they serve no real value since the JDBC integration test can be run locally without additional credentials.
3) the main value this class brings is letting us run tests without building the docker image (the integration tests require doing so). however I feel this benefit is not worth the additional maintenance cost.